docs/table/construct_table.rst

```diff
@@ -1244,19 +1244,13 @@ the ``copy`` preference of the original call to ``QTable()``::
 Here ``cls`` is the |Table| class or subclass that is being instantiated
 (|QTable| in this example), ``copy`` indicates whether a copy of the values in
 ``data`` should be provided, and ``**kwargs`` are any extra keyword arguments
-which are not valid |Table| ``_init_()`` keyword arguments. In the example
-above, ``strict_copy=True`` would end up in ``**kwargs`` and get passed to
+which are not valid |Table| ``__init__()`` keyword arguments. In the example
+above, ``mask_invalid=True`` would end up in ``**kwargs`` and get passed to
 ``__astropy_table__()``.
 
-If ``copy`` is `True` then the ``__astropy_table__()`` method must ensure that
-a copy of the original data is returned. If ``copy`` is `False` then a
-reference to the table data should be returned if possible. If it is not
-possible (e.g., the original data are in a Python list or must be otherwise
-transformed in memory) then ``__astropy_table__()`` method is free to either
-return a copy or else raise an exception. This choice depends on the preference
-of the implementation. The implementation might choose to allow an additional
-keyword argument (e.g., ``strict_copy`` which gets passed via ``**kwargs``) to
-control the behavior in this case.
+The ``copy`` argument should follow the conventions used by |Table|.
```

Comment:
I think we should just delete the paragraph altogether (i.e., also delete 1251-1253 now). We already say `copy` is passed through, and have just mentioned that `mask_invalid` ends up in `**kwargs`.

Reply:
good call. Fixed